sdk/scheduler/src/main/java/org/apache/mesos/specification/yaml/RawRLimit.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.mesos.specification.yaml;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Raw YAML individual rlimit specification.
+ */
+public class RawRLimit {
+    private Long soft;
```

Comment:
Same comment as `RawContainer`: Drop setters in favor of `@JsonCreator` constructor+`final`